ballerina/types.bal

```diff
@@ -46,6 +46,67 @@ public type Options record {
     boolean enableConstraintValidation = true;  
     # If `true`, when the result is a list it will contain headers as the first row.
     boolean outputWithHeaders = false;
+    # Specifies the fail-safe options for handling errors during processing.
+    # If enabled, metadata is printed to console/files based on the output mode configuration.
+    FailSafeOptions failSafe?;
+};
+
+# Represents the options for fail-safe mechanism during parsing.
+public type FailSafeOptions record {|
+    # Specifies enabling logging errors to the console
+    boolean enableConsoleLogs = true;
+    # Excludes logging source data to the console
+    boolean excludeSourceDataInConsole = true;
+    # Specifies the output mode for logging errors encountered during parsing
+    FileOutputMode fileOutputMode?;
+|};
+
+# Represents the file output mode for logging errors.
+public type FileOutputMode record {|
+    # The file path where errors will be logged
+    string filePath;
+    # Controls the level of detail included in the error logs.
+    ErrorLogContentType contentType = METADATA;
+    # Configuration for writing to the log file
```

Comment:
```suggestion
    # Configuration for writing to the error file
```

Reply:
Fixed in [bce4c69](https://github.com/ballerina-platform/module-ballerina-data.csv/pull/60/commits/bce4c69df8e43a3a06fce3ba6e2b7f5aec8d673b)